e2e/cypress/integration/enterprise/system_console/support_packet_generation_spec.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @system_console
+
+// # Goes to the System Scheme page as System Admin
+const goToAdminConsole = () => {
+    cy.apiAdminLogin();
+    cy.visit('/admin_console');
+};
+
+describe('Support Packet Generation', () => {
+    it('MM-T3817-Step1 - Commercial Support Dialog UI', () => {
```

Comment:
Please move `MM-T3817` to its own non-enterprise file: `e2e/cypress/integration/system_console/support_packet_generation_spec.js`

```suggestion
    it('MM-T3817 - Commercial Support Dialog UI - No License', () => {
```